cmd/cloud/utils.go

```diff
@@ -57,3 +57,15 @@ func runDryRun(request interface{}) error {
 	}
 	return nil
 }
+
+func createClient(flags clusterFlags) *model.Client {
+	if len(flags.headers) > 0 {
+		authHeaders := make(map[string]string, len(flags.headers))
+		for key, value := range flags.headers {
+			authHeaders[key] = value
+		}
+		return model.NewClientWithHeaders(flags.serverAddress, authHeaders)
```

Comment:
Technically the headers can be anything, so I would just use `headers` for the var name.
```suggestion
		headers := make(map[string]string, len(flags.headers))
		for key, value := range flags.headers {
			headers[key] = value
		}
		return model.NewClientWithHeaders(flags.serverAddress, headers)
```